### ISSUE TITLE
Option to colour roles based on team

### ISSUE THREAD

**Issue by Snorps** [TTT-2/TTT2#952]
It would be awesome if you could add an option to colour roles based on their team alignment rather than specific role colour. 
E.g Spy is green, Hitman is red, Jester remains pink
Just to make things a little more intuitive for new players.

Thanks!

**Comment by saibotk** [comment]
Should this act as an override to the default colors?

**Comment by TimGoll** [comment]
I see the problem with the current spy color. But the hitman is already read-ish?

**Comment by saibotk** [comment]
Yes, but i think this here is meant as a toggle to override all colors and only have the team color for all roles in the team, and i can see why some people would prefer that, especially with many roles.

**Comment by TimGoll** [comment]
understood. That would actually be pretty easy. We should add an accessability settings menu to the F1 menu that includes these types of settings
